renku_notebooks/api/schemas.py

```diff
@@ -155,6 +161,91 @@ class LaunchNotebookResponse(Schema):
     resources = fields.Nested(UserPodResources())
     image = fields.Str()
 
+    @pre_dump
+    def format_user_pod_data(self, server, *args, **kwargs):
+        """Convert and format a server object into what the API requires."""
+
+        def isoformat(dt):
+            """
+            Render a datetime object as an ISO 8601 UTC timestamp.
+            Naïve datetime objects are assumed to be UTC
+            """
+            if dt is None:
+                return None
+            dt = dt.astimezone(timezone.utc).replace(tzinfo=None)
+            return dt
```

Comment:
@ableuler the marshmallow schema takes care of this when it marshals the response. If I convert to string it actually errors out because it tries to call `.isoformat()` on the python `datetime`. So I just fully removed that whole function - we do not need it.

It knows to call `isoformat` because that field is defined like this in the schema:
```
started = fields.DateTime(format="iso", allow_none=True)
```